commune_app/models.py

```diff
@@ -1,3 +1,13 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+
+class User(AbstractUser):
+    id = models.IntegerField(unique=True, primary_key=True)
+    first_name = models.CharField(max_length=250)
+    last_name = models.CharField(max_length=250)
```

Comment:
```suggestion
    last_name = models.CharField(max_length=50)
```

Reply:
i think the expenses field should be a one to many field , cause for a signle user there is multiple expenses